src/models/point/priority_array.py

```diff
@@ -24,13 +24,29 @@ class PriorityArrayModel(db.Model):
     def __repr__(self):
         return f"PriorityArray(uuid = {self.point_uuid})"
 
+    def update(self, **kwargs):
+        for key, value in kwargs.items():
+            if hasattr(self, key):
+                setattr(self, key, value)
+        self.check_self()
+        db.session.commit()
+
+    def check_self(self) -> (bool, any):
+        if not self.get_highest_priority_value_from_priority_array(self):
+            from src.models.point.model_point import PointModel
+            point: PointModel = PointModel.find_by_uuid(self.point_uuid)
+            self._16 = point.fallback_value if point else 16
```

Comment:
```python3
            from src.models.point.model_point import PointModel
            point: PointModel = self.point
            self._16 = point.fallback_value
```
Currently, `backref` on `priority_array_write` is `points`, change it to `point` to make it work.